cmd/fetch_repo/go_mod_download.go

```diff
@@ -44,34 +44,28 @@ func runGoModDownload(dl *GoModDownloadResult, dest string, importpath string, v
 
 	cmd.Stdout = buf
 	cmd.Stderr = bufErr
-	fmt.Printf("Running: %s %s\n", cmd.Path, strings.Join(cmd.Args, " "))
 	dlErr := cmd.Run()
 	if dlErr != nil {
 		if _, ok := dlErr.(*exec.ExitError); !ok {
 			if bufErr.Len() > 0 {
-				return fmt.Errorf("!%s %s: %s", cmd.Path, strings.Join(cmd.Args, " "), bufErr.Bytes())
```

Comment:
`bufErr.Bytes()` will output bytes which is not intended for `%s`. Moving to `.String()`